src/frontend/src/models/Personnel.ts

```diff
@@ -0,0 +1,23 @@
+export enum ApiAccountStatus {
+  Available ="Available",
+  Invited = "Invited",
+  NoAccount = "NoAccount"
+}
+
+export type PersonnelDiscipline = {
+  name : string
+}
+
+type Personnel = {
+  AzureUniquePersonId : string,
+  Name:string,
+  JobTitle:string,
+  PhoneNumber:string,
+  Mail:string,
+  AzureAdStatus: ApiAccountStatus,
```

Comment:
Probably easier if this is just strings? Like `AzureAdStatus: 'Available' | 'Invited' | 'NoAccount',`